src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -61,6 +67,8 @@ export class EssentialsComponent extends BaseControllerDirective {
 
     this.infraCollection = this.data.infrastructureCollection;
     this.infraSettings = this.settings.getNewOrExistingInfrastructureSettings();
+    this.fmQuorumLossWarning = `The Failover Manager service is in quorum loss state. Cluster may not be responding. 
+    Service failover, automatic recovery will be blocked.Health/availability state of services/nodes may not be reflected.`
```

Comment:
The Failover Manager service is in quorum loss state, which can cause disruptions in the cluster. Operations like <list operations that are effected by this> will likely fail or get stuck. 

Word it more like that to make it clear which operations specifically will fail. Additionally do we have a public TSG for how to investigate FM in quorum loss? We should when providing that there is an issue also have guidance for how to handle it readily available

Reply:
There is an FM deadlock TSG: https://eng.ms/docs/cloud-ai-platform/azure-core/one-fleet-platform/one-fleet-platform-timmall/service-fabric-service/service-fabric-service/tsgs-and-traces-by-component/failover/tsg/fm-deadlock. I can discuss with Manoj to see if we have other public guidance on that.

Reply:
The above TSG does not apply to this situation. I didn't find existing source for guidance here. But we do know we can use client API to query the state of FM (FMM can still response to getPartition in this case). So I added a link in the text explaining about this. If needed I can add the link of TSG page that explains what quorum loss means and some basic FM concepts.

Reply:
my main point being if we can see this issue and present it to the customer we should be able to give some info on how to deal with it